controller/server/update.mli

```diff
@@ -26,4 +26,30 @@ type state =
   | ReinstallRequired
 [@@deriving sexp]
 
+type sleep_duration = float (* seconds *)
```

Comment:
I would generally either just use a float or, if defining a dedicated type, leave it abstract in the `.mli`.

```ocaml
type sleep_duration
```